src/preStartup.js

```diff
@@ -1,14 +1,22 @@
 import doesDatabaseVersionMatch from "@reactioncommerce/db-version-check";
 import { migrationsNamespace } from "../migrations/migrationsNamespace.js";
 
-const expectedVersion = 3;
+const expectedVersion = 4;
```

Comment:
Something doesn't sit right with having this expected version hardcoded in a source file. Is there a good reason we don't  look at the filesystem and determine the latest migration automatically? It would be on startup so the FS performance hit would be negligible and nonrecurring.

If there's a good reason for setting it manually, how do you feel about having it in a file that would contain just this number, or by putting it in package.json? At least then we could increment it automatically if we ever provided any sort of CLI tooling around migrations.

Reply:
@rosshadden It might be worth exploring some tooling around that. I feel like moving it prior to having a reason to move it would be just an extra file and premature optimization, though.